private-templates-service/client/src/components/Designer/Designer.tsx

```diff
@@ -60,18 +66,42 @@ interface State {
   isSaveOpen: boolean;
 }
 
+interface MatchParams {
+  uuid: string;
+  version: string;
+}
+
 let designer: ACDesigner.CardDesigner;
 
 class Designer extends React.Component<DesignerProps, State> {
   constructor(props: DesignerProps) {
     super(props);
     props.setPage(this.props.templateName, "Designer");
-    this.state = { isSaveOpen: false };
+    this.state = {isSaveOpen: false };
+    if(this.props.match.params.uuid !== "newcard"){
+      this.props.getTemplate(this.props.match.params.uuid);
+    }
   }
 
   toggleModal = () => {
     this.setState({ isSaveOpen: !this.state.isSaveOpen });
   }
```

Comment:
is this used anywhere? I didn't see it. If not line 80 `this.state = {isSaveOpen: false };` and the State interface can be removed as well.